src/components/dropdownSearch/DropdownSearchReducers.tsx

```diff
@@ -44,9 +50,28 @@ export const getNextIndexPosition = (array: any[], item: any, key: number): numb
   return index;
 };
 
+export const removeSelectedOption = (state: IDropdownSearchState, displayValue: string): FixedQueue<IDropdownOption> => {
+  return new FixedQueue<IDropdownOption>(_.filter(state.selectedOptions.getQueue(),
+    (selectedOption: IDropdownOption) => {
```

Comment:
The displayValue is more Reliable because in the case that we have a new custom option, it would only have a display value and a random value as a key